src/renderers/shaders/ShaderChunk/morphtarget_pars_vertex.glsl.js

```diff
@@ -20,12 +20,34 @@ export default /* glsl */`
 
 		vec4 getMorph( const in int vertexIndex, const in int morphTargetIndex, const in int offset ) {
 
-			int texelIndex = vertexIndex * MORPHTARGETS_TEXTURE_STRIDE + offset;
+			int texelIndex = vertexIndex * MORPHTARGETS_TEXTURE_STRIDE + 3 * offset;
```

Comment:
Why do we need to multiple the `offset` with `3` now?

Reply:
To keep in line with what we previously had on the shader chunks that call getMorph. Now the position is at offset 0, normals at offset 3 and colors at offset 9

Reply:
Let's keep the current version of this PR 👍 .